DCA/adapter_utils.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2022, Ilya Syresenkov, Kirill Ivanov and Anastasiia Kornilova
```

Comment:
Is this adapter utils? I think it is related to run execution, maybe simple rename it to utils